### DISCUSSION TITLE
E36D96BF-2030-47A5-8ADB-0C173DAA60A5

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#44] [General]
# E36D96BF-2030-47A5-8ADB-0C173DAA60A5

TOI-1136 is a young (700 Myr) G-type star located about 275.26 lightyears away from the solar system, 6 Neptunian planets (b= 1.90 Re, c=2.88 Re, d= 4.63 Re, e= 2.64 Re, f= 3.88 Re, g= 2.53 Re) orbit resonantly and close to their star. Radial Velocity and TTVs were used to record the masses of all these planets (b= 3.01 Me, c= 6.0 Me, d= 8.0 Me, e= 5.4 Me, f= 8.3 Me, g=4.8 Me) with some wide ranges for planets f and g. the system has a notable chain of resonant orbits which are listed below.
b:c = 3:2
c:d = 2:1
d:e = 3:2
e:f = 7:5
f:g = 3:2

https://celestia.mobi/addon?item=E36D96BF-2030-47A5-8ADB-0C173DAA60A5&lang=en

**Comment by exocritic5457** [comment]
TOI-1136 managed to beat TOI-178 at its own game. All 6 of its planets are in resonance, beating every other STIP with a resonance chain,  making it second only to the dreaded TRAPPIST-1. Phenomenal!

**Reply by exocritic5457** [reply]
turns out HD 110067 happened as well. That STIP's also got a nice 6-planet resonance, here's to another

**Comment by Karistus** [comment]
If anyone is having trouble finding the system, search for "TYC 4165-581-1", for some reason the star never got the TOI designation i assigned it
